packages/aws-serverless/src/sdk.ts

```diff
@@ -361,3 +371,19 @@ export function wrapHandler<TEvent, TResult>(
     });
   };
 }
+
+/**
+ * Checks if Otel's AWSLambda instrumentation successfully wrapped the handler.
+ * Check taken from @opentelemetry/core
+ */
+function isWrappedByOtel(
```

Comment:
figured I'd just pull in this function rather than declaring `opentelemetry/core` a dependency of aws-serverless. Happy to change if reviewers prefer it!